vald/start.go

```diff
@@ -380,9 +380,9 @@ func getStartBlock(cfg config.ValdConfig, stateStore StateStore, nodeHeight int6
 	return startBlock, nil
 }
 
-func createEventBus(client *tendermint.RobustClient, startBlock int64, logger log.Logger) *tmEvents.Bus {
-	notifier := tmEvents.NewBlockNotifier(client, logger).StartingAt(startBlock)
-	return tmEvents.NewEventBus(tmEvents.NewBlockSource(client, notifier, logger), pubsub.NewBus[tmEvents.ABCIEventWithHeight](), logger)
+func createEventBus(client *tendermint.RobustClient, startBlock int64, retries int, backOff time.Duration, logger log.Logger) *tmEvents.Bus {
+	notifier := tmEvents.NewBlockNotifier(client, logger, tmEvents.Retries(retries), tmEvents.BackOff(backOff)).StartingAt(startBlock)
+	return tmEvents.NewEventBus(tmEvents.NewBlockSource(client, notifier, logger, tmEvents.Retries(retries), tmEvents.BackOff(backOff)), pubsub.NewBus[tmEvents.ABCIEventWithHeight](), logger)
```

Comment:
Why do `retries` and `backOff` have to be passed in for both the block notifier and the block source?

Reply:
the notifier tells you there is a new block, the source is pulling it from the node. So it's two different calls and I didn't see a good reason why the parameters should be different for each.